Gaijinizer patch/Map105.txt

```diff
@@ -367,8 +388,8 @@ Wow......#MTLed
 > CONTEXT: Map105/events/16/pages/1/43/Dialogue/enemy7
 > CONTEXT: Map143/events/15/pages/1/39/Dialogue/enemy7
 > CONTEXT: Map143/events/16/pages/1/43/Dialogue/enemy7
-By the way...what is the key
-Why do you like this?#MTLed
+By the way... what's Mr Important doing wandering
+around in a place like this?#Ron
```

Comment:
Add a period after "Mr" for "Mr." here. Could also lengthen it to be Mister, your call.